### PR TITLE
Grants docs

### PR DESCRIPTION
**Pull request by cmagan**
# Description

Update grants.md file adding overview, evaluation process and safes structure
Added a file application.md with some information about the application process

there are still links in the text that must get updated

# Changes

- [ ] ...
- [ ] ...

### CODE REVIEW

File: docs/governance/grants/grants-program.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+  The evaluation will consider their track record, expertise in relevant technologies, and the ability to execute the project effectively.
+
+By considering these judgment criteria, the CoW Grants Program aims to ensure that grants awarded align with the objectives of CoW Protocol and contribute to its growth and development.
+Grants that demonstrate feasibility, have a positive impact on the ecosystem, provide realistic workloads, and are backed by competent individuals or teams will have higher chances of receiving funding and support.
+
+# Committee Members and Safe Control Structure
+
+As determined in [CIP-8](https://snapshot.org/#/cow.eth/proposal/0xdc641be107f139753cea051f1bacb8b74b915713a95306c3994f5e03e20d6bef), the CoW Grants Program Safe is controlled by two signers. Safe #3 is shared with the CoW DAO (Safe #1) and the Grants Committee (Safe #2). Safe #3 holds all the grant funds. You can check the governing process in the image below:
+
+(Todo: Create a mermaid chart of Safes including committee, grants-funds, volume-grants, cow-dao)
```
Comment (mfw78) [inline docs/governance/grants/grants-program.md]: Please insert the mermaid chart before having this merged (or remove the todo)

File: docs/governance/grants/grants-program.md [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+
+The grants committee will have a distribution of 8 community members, making a total of 7 committee members. Safe #2 (from the schematic image above) [`gno:0xCA1F000D520c21C47E6c634DD31b92b91A6338bD`](https://app.safe.global/settings/setup?safe=gno:0xCA1F000D520c21C47E6c634DD31b92b91A6338bD) will function as the Grants committee Safe and has 8 signers with a signing threshold of 5 out of 8. The specified committee members:
```
Comment (mfw78) [inline docs/governance/grants/grants-program.md]: Confirm the number of committee / community members

**Review by mfw78 — COMMENTED** [review]
Comments left in-line below. Also, for other reviewers, please have a once-over for grammar.

@cmagan Please sign the CLA per the comments above in the PR (just requires a simple "comment" to be made on your behalf)

**Comment by cmagan** [comment]
I have read the CLA Document and I hereby sign the CLA

File: docs/governance/grants/grants-program.md (hunk 2 above)
Comment (mfw78) [reply docs/governance/grants/grants-program.md]: Are there 8 community members?

Comment (cmagan) [reply docs/governance/grants/grants-program.md]: IMO this becomes increasingly irrelevant, we can just say the number of committee memebers

Comment (mfw78) [reply docs/governance/grants/grants-program.md]: If it's irrelevant, or seemingly irrelevant, we can remove it. It's just that "8 community members, making a total of 7 committee members" doesn't make sense?

**Comment by cmagan** [comment]
Hey @mfw78, I'm not sure I understand the remaining items to get this merged?

File: docs/governance/grants/application-process.md [hunk 3]
```diff
@@ -16,9 +16,23 @@ To apply for the CoW Grants program, use the [application template](application-
...
+3. To approve a grant application, at least 5 out of 8 committee members must vote in favor on the snapshot proposal.
 
 ### Funding
 
 1. After a successful snapshot vote, the committee will verify the proposed transaction for the Grants Program Safe and execute payment transactions based on achieved milestones.
 2. If milestones are not met, the Grants Committee reserves the right to withhold funding until milestones are completed, or to formally close the grant.
+
+To incorporate the adjustment you mentioned, here's a revised version of the clause that specifies the timeline is three months after the planned completion date as outlined in the grant proposal:
+
+---
```
Comment (mfw78) [inline docs/governance/grants/application-process.md]: Is this meant to be here?

**Review by mfw78 — COMMENTED** [review]
Ideally this should also be reviewed by @gogonimago or @marshymarsh as well for copy

**Review by mfw78 — APPROVED** [review]

**Pull request merged**